src/components/Download/Download.css

```diff
@@ -0,0 +1,19 @@
+.download-underline {
+  background-color: #08a045;
+  height: 2px;
+  width: 100%;
```

Comment:
it is a block level element , do we really need this ` width:100%`, 
i guess it is set by default ?

Reply:
I Will fix this in the next push